packages/app/app/[organization]/components/ChannelDescription.tsx

```diff
@@ -0,0 +1,50 @@
+'use client'
+import React, { useEffect, useRef, useState } from 'react'
+
+const ChannelDescription = ({
+  description,
+}: {
+  description?: string
+}) => {
+  const contentRef = useRef<HTMLDivElement>(null)
+  const [isClamped, setClamped] = useState(false)
+  const [isExpanded, setExpanded] = useState(false)
+
+  useEffect(() => {
+    function handleResize() {
+      if (contentRef && contentRef.current) {
+        setClamped(
+          contentRef.current.scrollHeight >
+            contentRef.current.clientHeight
+        )
+      }
+    }
+    handleResize()
+    window.addEventListener('resize', handleResize)
+
+    return () => window.removeEventListener('resize', handleResize)
+  }, [contentRef])
+
+  return (
+    <div className="hidden md:block w-4/5">
```

Comment:
Guess this is on purpose, but I do not see the description at all anymore on mobile

Reply:
Yes, I follow the previous logic of hiding the text on mobile, but we can reduce the font size if we want to have a description on mobile.